Sources/UI/Channels/ChannelsViewController.swift

```diff
@@ -123,8 +123,43 @@ open class ChannelsViewController: ViewController {
     
     // MARK: - Channel Cell
     
+    /// Dequeues and returns the channel cell at a given indexPath.
+    ///
+    /// If the dequeued cell is `ChannelTableViewCell` or is a subclass of it, `customizeChannelCell` function is called with it.
+    ///
+    /// This function should be overridden if one wants to customize their own cell class.
+    /// If your own cell class inherits from `ChannelTableViewCell`, simply register it in `tableView` as usual - you don't need to dequeue it manually.
+    /// You should override `updateChannelCell` function to configure your own `ChannelTableViewCell` subclass.
+    ///
+    /// - Parameter indexPath: indexPath to be dequeued
+    /// - Parameter channelPresenter: ChannelPresenter for the indexPath (see `ChannelPresenter`)
+    /// - Returns: Dequeued UITableViewCell for the given indexPath
     open func channelCell(at indexPath: IndexPath, channelPresenter: ChannelPresenter) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(for: indexPath) as ChannelTableViewCell
+        let channelCellIdentifier: String
+        if let channelTableView = tableView as? ChannelTableView {
```

Comment:
1. Please add an empty line before `if`.
2. It should be `guard`, else `return .empty`